util/util.go

```diff
@@ -33,7 +33,7 @@ var (
 
 func init() {
 	// init basic logger
-	logPath := fmt.Sprintf("/tmp/%s-debug.log", AppName)
+	logPath := fmt.Sprintf("%s/%s-debug.log", os.TempDir(),AppName)
```

Comment:
👋 For whatever it's worth, I have a drive by comment on this, from a user perspective...

Ultimately, I agree that `os.TempDir()` is preferable to a hard-coded `/tmp` directory. However, I'm a bit worried doing so -- given `e1s`'s current implementation -- would make `e1s-debug.log` prohibitively hard to find/use considering...

* `os.TempDir()` may return non-obvious values like `/var/folders/46/sz1dzp417x7fk46kb3jv8cn00000gp/T/`
* `e1s` often does not surface errors to users via the UI
* `e1s-debug.log` is often the only way users can get insight underlying errors

Ultimately, I think `e1s` should proactively handle/surface errors (and/or perhaps even allow its own logs and diagnostics to be browsed within its UI). However, that's a bigger effort :)

Alternatively, in the short term, could this PR add support for a `-log-file-path` option through which users could optionally define their own log file path, overwriting the `os.TempDir()` default?

If you do more forward with this PR, though, I think it warrants a `README.md` update: https://github.com/keidarcy/e1s/?tab=readme-ov-file#logs

Reply:
Also, disregarding what I suggest above and moving forward with your current vision, I think it ought to be the following, ensuring the `logPath` is correct across operating systems: 

```suggestion
	logPath := filepath.Join(os.TempDir(), fmt.Sprintf("%s-debug.log", AppName))
```

Reply:
I still need to iron out some wrinkles (and fix the tests!), but perhaps PR #127 illustrates some alternative ideas that address your concerns, @rromeroatleadtech ?